cli-wrap/src/main/resources/cli/listings.sh

```diff
@@ -257,24 +343,40 @@ pom_bw_junit_xml() {
 
 scm_report() {
     local type="pretty"
-    if [ "X$1" = "X--raw" ]; then
-        type="raw"
-        shift
-    elif [ "X$1" = "X--json" ]; then
-        type="json"
-        shift
-    fi
-
+    local productNames=""
+    local productVersionIds=""
     local scm="$1"
     local tag="$2"
     local pom_path="$3"
+    
```

Comment:
what about  ``shift 3`` after this, so you don't go over $1, $2 and $3 in the while loop?